packages/flutter/lib/src/gestures/scale.dart

```diff
@@ -99,11 +106,41 @@ bool _isFlingGesture(Velocity velocity) {
   return speedSquared > kMinFlingVelocity * kMinFlingVelocity;
 }
 
+
+/// Defines a line between two pointers on screen.
+///
+/// [_LineBetweenPointers] is an abstraction of a line between two pointers in
+/// contact with the screen. Used to track the rotation of a scale gesture.
+class _LineBetweenPointers{
+
+  /// Creates a [_LineBetweenPointers]. None of the [pointerStartLocation], [pointerStartId]
+  /// [pointerEndLocation] and [pointerEndId] must be null. [pointerStartId] and [pointerEndId]
+  /// should be different.
+  _LineBetweenPointers({
+    this.pointerStartLocation = Offset.zero,
+    this.pointerStartId = 0,
+    this.pointerEndLocation = Offset.zero,
+    this.pointerEndId = 1
+  }) : assert(pointerStartLocation != null && pointerEndLocation != null),
+       assert(pointerStartId != null && pointerEndId != null),
+       assert(pointerStartId != pointerEndId);
+
+  /// The location and the id of the pointer that marks the start of the line,
```

Comment:
these should use `//` not `///` since they are comments that apply to multiple fields so we would have to update them if we made the class public (that's totally fine, we just make sure to use `//` instead when doing that).

Also, the last character should be a period not a comma.